instaclone/src/components/commentsection/commentinput.js

```diff
@@ -8,20 +8,20 @@ let CommentInput = props => {
     );
 };
 
-addNewComment = (event, index) => {
```

Comment:
Hey, Jon. You may want to place your `addNewComment` logic inside your `commentSection.js`. Inside your `commentinput.js` is where you will build your form.